src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--31">
+      <div class="calendar__day calendar__day--1 start-day-mon"></div>
```

Comment:
The days of the calendar are hard-coded in the HTML, which is not maintainable. Instead, consider generating the HTML for these days using JavaScript.

Reply:
it's task for CSS advanced. we don't need use JS in this task